src/unittest/test.cpp

```diff
@@ -169,10 +169,16 @@ Markdown lol \\|\\|spoiler\\|\\| \\~\\~strikethrough\\~\\~ \\`small \\*code\\* b
 	set_test(READFILE, false);
 	std::string rf_test = dpp::utility::read_file(SHARED_OBJECT);
 	FILE* fp = fopen(SHARED_OBJECT, "rb");
-	fseek(fp, 0, SEEK_END);
-	size_t off = (size_t)ftell(fp);
-	fclose(fp);
-	set_test(READFILE, off == rf_test.length());
+	if (!fp) {
```

Comment:
this does not need to be changed - it expects that the file is in place and that you run the unit test from the correct location. e.g. `./unittest` not `./build/unittest` or similar

Reply:
@Jaskowicz1 can revert commit 7ef50c0bdf3d0f38798b220e00c8255f7be4166d